src/spatialdata/_core/data_extent.py

```diff
@@ -157,7 +188,15 @@ def _(e: SpatialData, coordinate_system: str = "global") -> BoundingBoxDescripti
 
 
 @get_extent.register
-def _(e: GeoDataFrame, coordinate_system: str = "global") -> BoundingBoxDescription:
+def _(
+    e: GeoDataFrame,
+    coordinate_system: str = "global",
+    has_images: bool = True,
```

Comment:
A suggestion, if possible I would remove these arguments from the single dispatch implementation of shapes, points and raster data, and I would leave it in the `SpatialData` implementation. I think that if you add a **kwags to the base declaration (the one for `SpatialData | SpatialElement`) and remove the arguments where not needed the syntax should be correct, please give it a try.

Reply:
Sure, thanks for the feedback!